www/src/Util/Timers.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Feels like this format for the header is supposed to be 

`{$description};dur={$duration}`